src/common/MarkdownAndHtmlToHtml.tsx

```diff
@@ -33,9 +33,15 @@ async function convertMarkdownAndHtmlMixToSanitizedHtml(
     .use(stringify)
     .process(markdown)
 
-  const html = italic(subSup(String(file)))
-  //unified() outputs the HTML within a <p>...</p>, we remove it
-  return DOMPurify.sanitize(html.slice(3, html.length - 4))
+  const html = DOMPurify.sanitize(italic(subSup(String(file))))
+  /*if there is no user defined wrapper (could be in markdown) unified defaults to
+    wrapping its output in <p>...</p> which is problematic for inline use
+   */
```

Comment:
Comments telling what the code does are not useful and often mean that the code is not clear enough. Consider extracting a method with a descriptive name.